src/g_game.c

```diff
@@ -478,9 +462,9 @@ void G_PrepTiccmd(void)
 
   if (mouselook)
   {
-    const double pitch = CalcMousePitch(mousey);
-    cmd->lookdir = CarryMousePitch(pitch);
-    localview.pitch = cmd->lookdir;
+    const int pitch = CalcMousePitch(mousey);
+    cmd->pitch = CarryMousePitch(pitch);
```

Comment:
Not sure if this is working properly. Should `CalcMousePitch` return `double`?

Reply:
I restored `double`, but I still don't get it, it seems `CalcMousePitch` always returns a zero fractional part. @ceski-1, please explain why we need the `CarryMousePitch` function?

Reply:
If you use a non-default mouse sensitivity it will leave a fractional part.

Reply:
Got it, thanks.